pages/index.js

```diff
@@ -1,69 +1,93 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import { useState, useEffect } from 'react';
+import { Stack, Button } from '@mui/material';
+import saveAs from 'file-saver';
+import SelectMenu from '../components/SelectMenu';
+import CustomTable from '../components/CustomTable';
+import loadCardsData from './api/data';
+import { FDDHeaderRow, LumpSumHeaderRow, menuItems } from '../helper';
+import Head from 'next/head';
+import styles from '../styles/Home.module.css';
 
 export default function Home() {
-  return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+  const [cardType, setCardType] = useState('');
+  const [data, setData] = useState([]);
+  const [headerRow, setHeaderRow] = useState([]);
+
+  useEffect(() => {
+    if (cardType !== '') {
+      loadCardsData(cardType).then(setData);
+      cardType === 'FDD'
```

Comment:
store 'FDD' in a constant or make an object like
const rateCardTypes = { FDD: "FDD", Lumpsum: "Lumpsum" }
if you're using typescript, then make enumerators